src/adapters/azure_functions_adapter.ts

```diff
@@ -0,0 +1,58 @@
+import { Severity, ILogger, Options } from "../logger"
+
+export enum AzureFunctionsSeverity {
+  Error = "error",
+  Warn = "warn",
+  Verbose = "verbose",
+  Info = "info",
+}
+
+export class AzureFunctionsAdapter implements ILogger {
+  name: string = "AzureFunctions"
+
+  minimumLogLevel: Severity
+
+  context: any
```

Comment:
it would require a direct dependency on the package so we prefer to avoid doing this now. We might improve it in the future by creating our own interface